src/main/java/com/gmail/nossr50/listeners/EntityListener.java

```diff
@@ -77,43 +109,40 @@ public void onEntityTransform(EntityTransformEvent event) {
 
     @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
     public void onEntityTargetEntity(EntityTargetLivingEntityEvent event) {
-        if(!ExperienceConfig.getInstance().isEndermanEndermiteFarmingPrevented())
+        if (!ExperienceConfig.getInstance().isEndermanEndermiteFarmingPrevented()) {
             return;
+        }
 
         //It's rare but targets can be null sometimes
-        if(event.getTarget() == null)
-        {
+        if (event.getTarget() == null) {
             return;
         }
 
         //Prevent entities from giving XP if they target endermite
-        if(event.getTarget() instanceof Endermite)
-        {
-            if(event.getEntity() instanceof Enderman) {
-                Enderman enderman = (Enderman) event.getEntity();
+        if (event.getTarget() instanceof Endermite 
+                && event.getEntity() instanceof Enderman) {
+            Enderman enderman = (Enderman) event.getEntity();
 
-                if(!persistentDataLayer.hasMobFlag(MobMetaFlagType.EXPLOITED_ENDERMEN, enderman)) {
-                    persistentDataLayer.flagMetadata(MobMetaFlagType.EXPLOITED_ENDERMEN, enderman);
-                }
+            if (!persistentDataLayer.hasMobFlag(MobMetaFlagType.EXPLOITED_ENDERMEN, enderman)) {
+                persistentDataLayer.flagMetadata(MobMetaFlagType.EXPLOITED_ENDERMEN, enderman);
             }
         }
     }
 
     @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
     public void onEntityShootBow(EntityShootBowEvent event) {
         /* WORLD BLACKLIST CHECK */
-        if(WorldBlacklist.isWorldBlacklisted(event.getEntity().getWorld()))
+        if (WorldBlacklist.isWorldBlacklisted(event.getEntity().getWorld())) {
             return;
+        }
 
-        if(event.getEntity() instanceof Player)
-        {
+        if(event.getEntity() instanceof Player) {
             Player player = (Player) event.getEntity();
 
             /* WORLD GUARD MAIN FLAG CHECK */
-            if(WorldGuardUtils.isWorldGuardLoaded())
-            {
-                if(!WorldGuardManager.getInstance().hasMainFlag(player))
-                    return;
+            if (WorldGuardUtils.isWorldGuardLoaded() 
+                    && !WorldGuardManager.getInstance().hasMainFlag(player)) {
+                return;
```

Comment:
Instead of unnesting them it would be better to have a method do both checks, the goal is readability.